src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Sanitizer.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Logging
+{
+    /// <summary>
+    /// Utility class for sanitizing logging strings.
+    /// </summary>
+    public static class Sanitizer
```

Comment:
Any objections to this being public? We'll need it in Script as well...

Reply:
FYI -- this came from internal code that's been used for a while.

Reply:
This is high-liability code making a best-effort to implement an impossible contract.   Therefore, I prefer non-public. If Script really needs this; then copy it? 


---
In reply to: [147281384](https://github.com/Azure/azure-webjobs-sdk/pull/1403#discussion_r147281384) [](ancestors = 147281384)